bot/exts/evergreen/catify.py

```diff
@@ -0,0 +1,47 @@
+import random
+from typing import Optional
+
+from discord.ext import commands
+
+from bot.constants import Cats
+
+
+class Catify(commands.Cog):
+    """Cog for the catify command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["ᓚᘏᗢify", "ᓚᘏᗢ"])
+    async def catify(self, ctx: commands.Context, *, text: Optional[str]) -> None:
+        """
+        Convert the provided text into a cat themed sentence.
+
+        If no text is given then the users nickname is edited.
+        """
+        if not text:
+            display_name = ctx.author.display_name
+            if len(display_name) >= 28:
```

Comment:
iirc the max nickname length is 32 characters. Since the longest cat is 3 chars long, and we're adding `" | "` too, this should check that the name isn't longer than 32-6 (26) rather than 28.

Reply:
Resolved as of https://github.com/python-discord/sir-lancebot/pull/694/commits/c548057ffa27542d73404af25a757bee992f6feb